components/status.tsx

```diff
@@ -8,9 +8,25 @@ export default async function Status({
   description: string;
 }) {
   return (
-    <article className="m-auto max-w-fit text-center">
-      <h2>{title}</h2>
-      <p>{description}</p>
+    <article className="flex items-center justify-center h-screen bg-[#FDF8F2] text-[#DB4437]">
```

Comment:
```suggestion
    <article className="flex items-center justify-center h-screen text-primary-500">
```

we have premade font colors, use that instead of hardcoding a specific value